apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v1/coverageSearchByPatientId.json

```diff
@@ -8,7 +8,7 @@
   "total" : 4,
   "link" : [ {
     "relation" : "self",
-    "url" : "https://localhost:IGNORED_FIELD/v1/fhir/Coverage?_format=json&beneficiary=Patient%2F567834"
+    "url" : "https://localhost:IGNORED_FIELD1/v1/fhir/Coverage/?beneficiary=567834"
```

Comment:
Why did all of these change to `IGNORED_FIELD1`?

Reply:
Hmm. I mentioned this and other discrepancies in the purportedly generated endpoint-responses to @meliGuzman, and she pointed me in the right direction. Without any changes here to `EndpointJsonComparatorBase`, we must assume that these are only _partially_ generated and manually updated to include these bespoke `IGNORED_FIELD1` values.

Along the same lines, is that what's occurring with the rest of the query string?